setup.ts

```diff
@@ -30,3 +31,20 @@ await log.setup({
     },
   },
 });
+
+let connected = false;
+
+if (Deno.env.get("PLS_CONNECTION_URI")) {
```

Comment:
Just to clear myself, how this ENV var will be used?

Reply:
By the surprise 😎